paystack/src/main/java/co/paystack/android/api/model/ChargeResponse.kt

```diff
@@ -0,0 +1,44 @@
+package co.paystack.android.api.model
+
+
+import androidx.annotation.Keep
+import com.google.gson.Gson
+import com.google.gson.annotations.SerializedName
+
+@Keep
+data class ChargeResponse(
+
+    val status: String?,
+
+    @SerializedName("trans")
+    val transactionId: String,
+
+    val reference: String,
+
+    val message: String?,
+
+    @SerializedName("otpmessage")
+    val otpMessage: String? = null,
+
+    val auth: String? = null,
+
+    @SerializedName("countryCode")
+    val countryCode: String? = null,
+
+    ) {
+
+    companion object {
+        fun fromJsonString(jsonString: String?): ChargeResponse {
+            return try {
+                Gson().fromJson(jsonString, ChargeResponse::class.java)
```

Comment:
It might be worth having a singleton reference to Gson, so that multiple Gson objects are not created when multiple calls to ```fromJsonString()``` are made.